Formula/mesos.rb

```diff
@@ -86,6 +54,13 @@ def install
 
     ENV.cxx11
 
+    # macOS no longer provides JavaVM.framework. Fix broken configure script
+    # that assumes this framework exists for JNI linking.
+    ENV["JAVA_CPPFLAGS"] = "-I#{Formula["openjdk@11"].include}"
+    ENV["JAVA_HOME"] = Language::Java.java_home("1.8")
```

Comment:
```suggestion
    ENV["JAVA_HOME"] = Language::Java.java_home("11")
```
?

Reply:
I'm rolling back the openjdk8 to 11 change since I haven't gotten it to build locally yet and want to reduce the diff if possible. We can explore simplifying it and updating openjdk versions if it builds on CI.